### DISCUSSION TITLE
Video demo of using Docker for Pi Pico SDK project building

### DISCUSSION THREAD

**Discussion by shabaz123** [lukstep/raspberry-pi-pico-docker-sdk#6] [General]
Hi Lukasz,

Thank you for your work. Using a Docker Container for Pi Pico development will save a lot of time for users. I found it useful, so I created a video demonstrating the steps.

[YouTube: Simplified Pi Pico C SDK Project Building](https://www.youtube.com/watch?v=CYBmfYR3Ti8)

**Comment by lukstep** [comment]
Thanks @shabaz123  for your video, I'm delighted this repo helps Pico developers.

If You looking for a better way to upload the binary to RPi Pico, you need to try to use the Pico debug probe with OpenOCD. Right now I'm testing this solution on Mac and works well.
